### ISSUE TITLE
How do I extend an existing measure, but also still be able to use all of the rest of convert-units from the same script?

### ISSUE THREAD

**Issue by sglindme** [convert-units/convert-units#299]
I have an api that converts units. I want to extend the length measure to add in some custom length units, but I still need that api function to be able to convert other units (like volume). 

It looks like in the example in the readme, this just creates a new convert variable that only has length:

```
const convert = configureMeasurements<'length', LengthSystems, NewLengthUnits>(
  { length: extendedLength }
);
```

I could be wrong. I can't test the example because it gives an error: "Cannot find module 'convert-units/definitions/length' or its corresponding type declarations.ts(2307)"

I'm on 3.0.0@beta5

**Comment by Taar** [comment]
I just published a new release 3.0.0-beta.6, try that and let me know if you're still having issues.

**Comment by Taar** [comment]
Going to assume that the latest version fixed this issue. If not, please create a new issue.
